content/docs/fundamentals/converting/resolving-errors@ko.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: 유효성 검사 오류 해결
+$order: 2
+toc: true
+---
+
+[TOC]
+
+이 섹션에서는 AMP 페이지에 있는 AMP 유효성 검사 오류 및 해결 방법을 살펴봅니다.  콘솔에 따라 오류가 아래와 다른 순서로 나타날 수 있습니다.
+
+## 문자 집합 포함
+
+우선 다음 오류를 해결해 보겠습니다.
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).